lib/components/tabs.vue

```diff
@@ -6,8 +6,11 @@
                     <a :class="['nav-link',{small: small, active: tab.localActive, disabled: tab.disabled}]"
                        :href="tab.href"
                        @click.prevent.stop="setTab(index)"
-                       v-html="tab.title"
-                    ></a>
+                       v-if="!tab.headHtml"
+                       >{{ tab.title }}</a>
+                    <div :class="['tab-head',{small: small, active: tab.localActive, disabled: tab.disabled}]"
```

Comment:
I've tested this construction `<a :is="headHtml?'a':'div'>`, but it doesn't work.
The 2 reasons for using the second case with <div> are:
1) to be able to customize the tab head
2) to be able to make the tab head not clickable

If <a :is="headHtml?'a':'div'> works the tab head still have @click.

Because of the reasons above i will keep the change as i've proposed it.